app/service-worker.js

```diff
@@ -37,8 +37,11 @@ self.addEventListener('fetch', event => {
 	// don't try to handle e.g. data: URIs
 	if (!url.protocol.startsWith('http')) return;
 
+	// ignore dev server requests
+	if (url.hostname === self.location.hostname && url.port !== self.location.port) return;
```

Comment:
at the moment the host for the dev client is always `localhost:${someport}` — it should probably be `${window.location.hostname}:${someport}` so that debugging on mobile devices etc works